### PR TITLE
Feature/add check user id logic into command

### PR DESCRIPTION
**Pull request by borovHokage**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/815

### CODE REVIEW

File: src/TimeService.Business/Commands/LeaveTime/CreateLeaveTimeCommand.cs [hunk 1]
```diff
@@ -47,8 +90,16 @@ public OperationResultResponse<Guid> Execute(CreateLeaveTimeRequest request)
 
             _validator.ValidateAndThrowCustom(request);
 
+            var existUsers = CheckUserExistence(userIds, errors);
```
Comment (BelehovEgor) [inline src/TimeService.Business/Commands/LeaveTime/CreateLeaveTimeCommand.cs:93]: It is very strange logic here. You send one id, if response contains this, for what do you send to mapper userid in request and array with one element

Comment (BelehovEgor) [reply src/TimeService.Business/Commands/LeaveTime/CreateLeaveTimeCommand.cs:93]: Move check user existence  to validator

**Review by BelehovEgor — CHANGES_REQUESTED** [review]

**Review by MariyaSin — APPROVED** [review]

File: src/TimeService.Validation/LeaveTime/CreateLeaveTimeRequestValidator.cs [hunk 2]
```diff
@@ -1,19 +1,57 @@
...
+
+        private bool CheckUserExistence(List<Guid> userIds)
         {
+            string logMessage = "Cannot check existing users {userIds}";
+
+            try
+            {
+                IOperationResult<ICheckUsersExistence> response = _rcCheckUsersExistence.GetResponse<IOperationResult<ICheckUsersExistence>>(
+                    ICheckUsersExistence.CreateObj(userIds)).Result.Message;
+                if (response.IsSuccess && response.Body.UserIds.Count == 1)
```
Comment (BelehovEgor) [inline src/TimeService.Validation/LeaveTime/CreateLeaveTimeRequestValidator.cs:29]: maybe you should add check that id in request and in response are equal

**Review by BelehovEgor — APPROVED** [review]

File: src/TimeService.Business/Commands/LeaveTime/CreateLeaveTimeCommand.cs (hunk 1 above)
Comment (borovHokage) [reply src/TimeService.Business/Commands/LeaveTime/CreateLeaveTimeCommand.cs]: Fixed

**Review by ndoriomedov — APPROVED** [review]

File: src/TimeService.Validation/LeaveTime/CreateLeaveTimeRequestValidator.cs (hunk 2 above)
Comment (borovHokage) [reply src/TimeService.Validation/LeaveTime/CreateLeaveTimeRequestValidator.cs]: done

**Review by VLiamin — APPROVED** [review]

**Pull request merged**